packages/turf-concave/package.json

```diff
@@ -74,6 +74,7 @@
     "@turf/invariant": "^6.3.0",
     "@turf/meta": "^6.3.0",
     "@turf/tin": "^6.3.0",
-    "topojson": "3.x"
+    "topojson-client": "3.x",
```

Comment:
Do you have a preference here @mfedderly for importing package version numbers?

Reply:
Its my habit to just always use ^ or ~ for the simple ranges, but if people want to use .x that's fine too.